railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -91,7 +91,7 @@ def ensure_credentials_have_been_added
 
         def change_credentials_in_system_editor
           credentials.change do |tmp_path|
-            system("#{ENV["EDITOR"]} #{tmp_path}")
+            system(ENV["EDITOR"], tmp_path)
```

Comment:
Ah, I didn't realize `tmp_path` is a `Pathname`.  Adding a `to_s` should fix the build:

```suggestion
            system(ENV["EDITOR"], tmp_path.to_s)
```